frontend/src/static/js/utils/ramp_colour_generator.js

```diff
@@ -0,0 +1,65 @@
+function getRandomHex() {
```

Comment:
There are no changes in the code in this file right? Except what is required for the integration.

Reply:
There are no changes besides removing `this.` from the function calls and loading them into window.